charts/pipelines-library/templates/pipelines/autotests/gradle/github-build-edp.yaml

```diff
@@ -11,6 +11,7 @@ metadata:
     app.edp.epam.com/pipelinetype: build
     {{- include "edp-tekton.labels" $ | nindent 4 }}
 spec:
+  description: "The pipeline for {{ $k }} aut build."
```

Comment:
what is 'aut'? please check example for gerrit

Reply:
updated